reconcile/utils/terraform_client.py

```diff
@@ -519,8 +521,9 @@ def _is_ignored_rds_modification(self, account_name: str, resource_name: str,
         ]
         if len(changed_terraform_args) == 1 \
                 and 'engine_version' in changed_terraform_args:
+            region_name = get_region_from_availability_zone(before['availability_zone'])
             response = \
-                self._aws_api.describe_rds_db_instance(account_name, resource_name)
+                self._aws_api.describe_rds_db_instance(account_name, resource_name, region_name=region_name)
```

Comment:
Line too long, even for Black standards. Let's configure pylint to whine on these.

Reply:
I wonder if it's just time to remove the excludes unless someone on the team objects strongly because they're in the middle of a large refactor (I'm guessing not the case). Any thoughts @fishi0x01?